Resources/Prototypes/_Pirate/Catalog/Fills/Lockers/interdyne.yml

```
@@ -0,0 +1,169 @@
+# Med Interdyne
+- type: entity
+  parent: LockerInterdyneCommand
+  id: LockerMedPersonalInterdyneFilled
+  name: Шафа з медичним обладнанням Інтердайна
+  components:
+  - type: StorageFill
+    contents:
+      - id: MedicatedSuture
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)